src/sage/topology/simplicial_set_constructions.py

```diff
@@ -1222,29 +1230,31 @@ def __init__(self, maps=None, vertex_name=None):
             sage: T = T.unset_base_point()
             sage: CT = T.cone()
             sage: inc = CT.base_as_subset().inclusion_map()
-            sage: P = T.pushout(inc, inc)
-            sage: P.homology()
+            sage: P = T.pushout(inc, inc)                                               # optional - sage.graphs
+            sage: P.homology()                                                          # optional - sage.graphs
             {0: 0, 1: 0, 2: Z x Z, 3: Z}
-            sage: len(P.nondegenerate_simplices())
```

Comment:
No `sage.modules` or `sage.groups`?

Reply:
A lot of `sage.modules` are missing in this file. Adding now

Reply:
I meant L1226.